messages/EventKeyTemplates.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
 	"info": {
 		"categoryImplicitlyRun": "Implicitly including %s rules from category '%s'",
-		"jarAndXmlProcessed": "XML files collected from JAR [%s]: %s",
+		"jarAndXmlProcessed": "Cataloger: XML files collected from JAR [%s]: %s",
```

Comment:
Rather than attempting to remove these logs altogether for `scanner:run:dfa`, I opted to prefix them with `Cataloger`, to make it clear where they're coming from.